src/main/scala/ai/datahunters/md/config/processing/ProcessingConfig.scala

```diff
@@ -23,6 +24,7 @@ object ProcessingConfig {
   val ColumnsNamingConventionKey = "output.columns.namingConvention"
   val IncludeMetadataContentKey = "output.columns.includeMetadataContent"
   val MandatoryTagsKey = "filter.mandatoryTags"
+  val HashListKey = "hash.types.list"
```

Comment:
@dawidsau Basically we have the following main config: categories: output, input, filter, processing, enrichment. Please put it under processing, e.g.: `processing.hash.types`